config/routes.js

```diff
@@ -104,4 +104,7 @@ module.exports.routes = {
   '/test/loadSeasonFixture': 'TestController.loadSeasonFixture',
   '/test/loadMatchFixtures': 'TestController.loadMatchFixtures',
   '/test/loadFinishedGameFixtures': 'TestController.loadFinishedGameFixtures',
+
+  // Catch all for Vue paths
+  '/*': { controller: 'ViewController', action: 'serveIndex', skipAssets: true },
```

Comment:
This is required for sails to properly server the vue index, including any and all paths that aren't listed above functioning as a catch all.